### PR TITLE
Added security, auth controller, dtos for user login, user mapper

### PR DESCRIPTION
**Pull request by IvanFadieiev**

### CODE REVIEW

File: src/main/resources/db/changelog/db.changelog-master.yaml [hunk 1]
```diff
@@ -10,4 +10,8 @@ databaseChangeLog:
   - include:
       file: db/changelog/changes/02.10.2023.CreateRentalsTable.yaml
   - include:
-      file: db/changelog/changes/02.10.2023.CreateRentalsConnections.yaml
\ No newline at end of file
+      file: db/changelog/changes/02.10.2023.CreateRentalsConnections.yaml
+  - include:
+      file: db/changelog/changes/02.10.2023.create-payment-table.yaml
```
Comment (CodeWebWeaver) [inline src/main/resources/db/changelog/db.changelog-master.yaml]: Naming mismatch

File: src/main/java/com/jvavateam/carsharingapp/repository/user/UserRepository.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+import com.jvavateam.carsharingapp.model.User;
+import java.util.Optional;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface UserRepository extends JpaRepository<User, Long> {
+    @Query("FROM User u LEFT JOIN FETCH u.roles WHERE u.email = :email")
```
Comment (Dimagaa) [inline src/main/java/com/jvavateam/carsharingapp/repository/user/UserRepository.java]: Use @EntityGraph instead Fetch

**Review by Dimagaa — CHANGES_REQUESTED** [review]
Pretty Good! Just one fix to done

**Review by Dimagaa — APPROVED** [review]
Well Done!

**Review by CodeWebWeaver — APPROVED** [review]

**Pull request merged**